product_code_unique/tests/test_code_unique.py

```diff
@@ -1,6 +1,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
This file don't belong to this module

Reply:
It fixes CI

Reply:
@HviorForgeFlow Could you do it in another PR or at least in another commit ? Thanks

Reply:
Sure! @rousseldenis